design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -0,0 +1,361 @@
+type Query {
+
+    getNegotiableQuote(
+        id: ID!
+    ): NegotiableQuote @doc(description:"Get negotibale quote")
+
+    getAllNegotiableQuotes: [NegotiableQuote]! @doc(description: "Get all negotiable quotes")
+}
+
+type Mutation {
+
+    updatedPrices(
+        updatedQuotes: UpdatedPricesInput!
+    ): UpdatedPricesOutput! @doc(description:" This call refreshes item prices, taxes, discounts, cart rules in the negotiable quote. Quotes that are locked for the seller will not be updated.")
+
+    requestQuote(
+        negotiableQuote: NegotiableQuoteRequestInput!
+    ): RequestQuoteOutput! @doc(description:"Generates negotiable quote request")
+
+    assignBillingAddressToNegotiableQuote(
+        billingAddress: NegotiableQuoteBillingAddressInput!
+    ): AssignBillingAddressToNegotiableQuoteOutput @doc(description:"Assigns new billing address to negotiable quote")
+
+    estimateShippingCostForNegotiableQuote(
+        shippingAddress: NegotiableQuoteEstimateShippinAddressInput!
+    ): EstimateShippingCostForNegotiableQuoteOutput @doc(description:"Estimate shipping costs for a negotiable quote")
+
+    estimateShippingCostByAddressId(
+        shippingAddressId: NegotiableQuoteEstimateShippinAddressIdInput!
+    ): EstimateShippingCostByAddressIdOutput @doc(description:"Estimate shipping costs for a negotiable quote by Address ID")
+
+    estimateShippingInformationForNegotiableQuote(
+        addressInformation: NegotiableQuoteShippingInformationInput!
+    ): EstimateShippingInformationForNegotiableQuoteOutput @doc(description:"Get list of payment options and the order totals by address and shipping info")
```

Comment:
How about having `available_payment_methods` in the negotiable quote query response? Like we have in cart query.

Reply:
Added `available_payment_methods` to negotiable quote query response.

Reply:
There is actually a typo in my previous comment. I meant to say, how about introducing `available_shipping_methods` instead of `estimateShippingInformationForNegotiableQuote` query.

Reply:
added `available_shipping_methods`.